src/core/unit.c

```
@@ -6626,6 +6628,41 @@ static uint64_t unit_get_cpu_weight(Unit *u) {
         return cc ? cgroup_context_cpu_weight(cc, manager_state(u->manager)) : CGROUP_WEIGHT_DEFAULT;
 }
 
+void unit_get_exec_quota_stats(Unit *u, ExecContext *c, ExecDirectoryType dt, uint64_t *usage, uint64_t *limit) {
```

it's fine ignoring errors sometimes, but we prefer if that happens in the caller. i.e. i'd prefer if this would return an `int` value which can indicate success. The caller then can choose to ignore it (and quite likely all users in our codebase will), but that really should be up to the caller